sdk/ai/azure-ai-inference/azure/ai/inference/aio/_operations/_operations.py

```diff
@@ -1,4 +1,3 @@
-# pylint: disable=too-many-locals
```

Comment:
I forgot to tell you (and add it to my OneNote): I did end up suppressing quality gates errors coming from some of the auto-generated files. I don't remember of this was one of them. You may need to add them back.